website/pages/docs/commands/volume/register.mdx

```diff
@@ -82,14 +82,16 @@ context {
   single-node modes. Consult the documentation of the storage provider
   and CSI plugin.
 
-- `attachment_mode` `(string: <required>)` - The storage API that will
-  be used by the volume. Most storage providers will support
-  `"file-system"`, to mount pre-formatted file system volumes. Some
-  storage providers will support `"block-device"`, which will require
-  the job be configured with appropriate mount options.
-
-- `mount_options` <code>([mount_options][]:nil)</code> - Options for
-  mounting `block-device`volumes without a pre-formatted file system.
+- `attachment_mode` `(string: <required>)` - The storage API that will be used
+  by the volume. Most storage providers will support `"file-system"`, to mount
+  volumes using the CSI filesystem API. Some storage providers will support
+  `"block-device"`, which mount the volume with the CSI block device API
+  within the container.
+
+- `mount_options` <code>([mount_options][]:nil)</code> - Options for mounting
```

Comment:
not sure what the `[mount_options]` bit inside the parentheses is supposed to convey... it occurs below as well.

Reply:
Good catch... that's in the existing docs but it doesn't render as any kind of link. Will fix.